docs/api-inference/tasks/text-to-image.md

```diff
@@ -0,0 +1,103 @@
+## Text-to-image
+
+Generate an image based on a given text prompt.
+
+<Tip>
+
+For more details about the `text-to-image` task, check out its [dedicated page](https://huggingface.co/tasks/text-to-image)! You will find examples and related materials.
+
+</Tip>
+
+### Recommended models
+
+- [black-forest-labs/FLUX.1-dev](https://huggingface.co/black-forest-labs/FLUX.1-dev): one of the most powerful image generation models that can generate realistic outputs.
+- [latent-consistency/lcm-lora-sdxl](https://huggingface.co/latent-consistency/lcm-lora-sdxl): a powerful yet fast image generation model.
+- [Kwai-Kolors/Kolors](https://huggingface.co/Kwai-Kolors/Kolors): text-to-image model for photorealistic generation.
+- [stabilityai/stable-diffusion-3-medium-diffusers](https://huggingface.co/stabilityai/stable-diffusion-3-medium-diffusers): a powerful text-to-image model.
+
+This is only a subset of the supported models. Find the model that suits you best [here](https://huggingface.co/models?inference=warm&pipeline_tag=text-to-image&sort=trending).
+
+### API specification
+
+#### Inputs
+
+| Payload |   |    |
+| :--- | :--- | :--- |
+| **inputs** | _string, required_ | The input text data (sometimes called "prompt"). |
```

Comment:
Do we want to specify limits? E.g `num_inference_steps` can't be larger than N, etc

Reply:
We could be it can get tricky. I think for a v1 it's fine not to have them.

We already discussed this when writing the specs. In theory it's possible to define lower and upper bound in a json schema. But in practice, the limits can depend on the model or even on the framework powering the model (example [here](https://github.com/huggingface/huggingface_hub/pull/1968#issuecomment-1903542670) with `top_p` that can be 1.0 in transformers but not in TGI). That being said, most integer/number values have fixed limits.